internal/provider/pod.go

```diff
@@ -406,32 +406,37 @@ func unitPrefix(namespace, podName string) string {
 	return prefix + separator + namespace + separator + podName
 }
 
-func Image(name string) string {
-	el := strings.Split(name, separator) // assume well formed
+// Name returns <namespace>.<podname> from a well formed name.
+// Units are named as 'systemk.<namespace>.<podname>.<image> .
```

Comment:
Shouldn't it be `systemk.<namespace>.<pod>.<container>`? Cause Kubernetes guarantees unique container names per Pod but not images which can repeat in multiple containers of a same Pod.

Reply:
While I'm backing down on my other comments, it seems we agree this needs to be addressed.

Reply:
yep done